src/services/rollouts.js

```diff
@@ -177,12 +177,7 @@ module.exports = class RolloutsHelper {
 				tenant_code: tenant_code,
 			}
 
-			let rollout
-			if (getResourceData) {
-				rollout = await rolloutQueries.findOne(filter, {}, true) // include resourceDetails
-			} else {
-				rollout = await rolloutQueries.findOne(filter) // plain query
-			}
+			const rollout = await rolloutQueries.findOne(filter, {}, getResourceData)
```

Comment:
@adithyadinesh0412 Can u check this with @praveenKDass   as my understaning when he get some issue he added this